### PR TITLE
Count DOH events for each user only once in ASN aggregates

### PR DESCRIPTION
**Pull request by scholtzan**
This should fix the issue that we count only whether clients report enable/disable DOH events, instead of the number of reported events.

### CODE REVIEW

File: sql/telemetry_derived/asn_aggregates_v1/query.sql [hunk 1]
```diff
@@ -105,6 +105,13 @@ events_with_asn AS (
     asn_ip_address_range
   USING
     (network_ip, mask)
+  GROUP BY
+    submission_date,
+    client_id,
+    canary,
+    event_category,
+    event_object,
+    autonomous_system_number
```
Comment (scholtzan) [inline sql/telemetry_derived/asn_aggregates_v1/query.sql:114]: Grouping them here should ensure that for each client only up to one doh event will be aggregated in the final part of the query.

Comment (scholtzan) [reply sql/telemetry_derived/asn_aggregates_v1/query.sql:114]: This way `n_clients` would still refer to the total number of distinct client IDs part of a specific ASN

Comment (tdsmith) [reply sql/telemetry_derived/asn_aggregates_v1/query.sql:114]: It's a little more concise to write SELECT DISTINCT at line 86.

**Comment by scholtzan** [comment]
CI is currently failing due to https://github.com/mozilla/bigquery-etl/pull/887

**Pull request merged**